src/pages/TeamOverview/TeamOverview.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {useLocation, useParams} from 'react-router-dom';
+import {ListItem, UserData} from 'types/types';
+import {getUserColumns} from 'utils/utils';
+import {getTeamOverview, getUserData} from '../../api/api';
+import Card from '../../components/Card/Card';
+import {GlobalContainer} from '../../components/global.styled';
+import Header from '../../components/Header/Header';
+import List from '../../components/List/List';
+
+const getUsersCards = (users: UserData[]): ListItem[] => {
+    return users.map((user) => {
+        var columns = getUserColumns(user);
+        
```

Comment:
`getUserColumns` was created as an util method, because it was something used in more than one component.